common/k8stest/util_testpods.go

```diff
@@ -1218,3 +1218,70 @@ func GetMayastorPodNameonNodeByPrefix(prefix string, nodeName string) (string, e
 	}
 	return "", fmt.Errorf("failed to get mayastor pod with prefix %s on node %s", prefix, nodeName)
 }
+
+// GetCsiNodePodNameOnNodeByPrefix return csi node pod on a node with a given prefix
+func GetCsiNodePodNameOnNodeByPrefix(nodeName string) (string, error) {
```

Comment:
Could be made more general purpose as in `GetPodNameOnNodeByPrefix(nodeName string, prefix string)`
And then caller supplies `e2e_config.GetConfig().Product.CsiDaemonsetName` as `prefix`

Reply:
made it more generic https://github.com/openebs/openebs-e2e/compare/97f6bb9f1285bb8f08fe736bb498948de52c0b3f..7a4e8d1886a4b1bebeba350d30dde9c35e2ec096